examples/models/llama2/source_transformation/sdpa.py

```diff
@@ -99,8 +99,10 @@ def forward(
         k, v = self.kv_cache.update(input_pos, k, v)
         attn_mask = mask[None, None, input_pos]
 
-        k = k.repeat_interleave(self.n_rep, dim=1)
-        v = v.repeat_interleave(self.n_rep, dim=1)
+        if self.n_rep > 1:
```

Comment:
what's the reason to change it?

Reply:
I found these 2 ops being redundant when `self.n_rep == 1`. 

I only tested static-shape case, though.

Reply:
Ah yeah I guess you're testing llama2. These lines are needed for group query attention like llama3..

Reply:
Reverted the `llama_transformer.py` and `sdpa.py` changes